### PR TITLE
operators fetch the price of an asset from both chainlink and dia oracle

### PR DESCRIPTION
**Pull request by JdejesusIsaac**

### CODE REVIEW

File: config-files/operator-1/raft.db
Comment (Robert-H-Leonard) [inline config-files/operator-1/raft.db:1]: Could you remove this file from the pull request? These are log files to store messages sent via the raft protocol. We should probably add this to the .gitignore.

File: config-files/operator-2/raft.db
Comment (Robert-H-Leonard) [inline config-files/operator-2/raft.db:1]: Could you remove this file from the pull request? These are log files to store messages sent via the raft protocol. We should probably add this to the .gitignore.

File: config-files/operator-3/raft.db
Comment (Robert-H-Leonard) [inline config-files/operator-3/raft.db:1]: Could you remove this file from the pull request? These are log files to store messages sent via the raft protocol. We should probably add this to the .gitignore.

File: contracts/script/IncredibleSquaringDeployer.s.sol [hunk 1]
```diff
@@ -147,6 +147,7 @@ contract IncredibleSquaringDeployer is Script, Utils {
         erc20Mock = new ERC20Mock();
         priceAdapter = new PriceFeedAdapter();
         priceAdapter.addFeed("btc/usdc", address(0x1b44F3514812d835EB1BDB0acB33d3fA3351Ee43));
+        //priceAdapter.getPriceDia("BTC/USD");
```
Comment (Robert-H-Leonard) [inline contracts/script/IncredibleSquaringDeployer.s.sol]: Should this be comments out? I think this is needed so when we start our local anvil node the priceAdapter is in the right state.

File: operator/priceFSM.go [hunk 2]
```diff
@@ -215,6 +214,16 @@ func (f *fsm) Apply(l *raft.Log) interface{} {
...
+	//fetch dia price
+	diaPrice, err := f.priceFeedAdapter.GetPriceDia(&bind.CallOpts{}, request.FeedName)
+
+	if err != nil {
+		f.logger.Printf("Failed to fetch price", "err", err)
+		return nil
+	
+	}
+	diaResponse := PriceUpdateTaskResponse{Price: uint32(diaPrice.Uint64()), Source: "dia", TaskId: request.TaskId, Decimals: 18}
+
```
Comment (Robert-H-Leonard) [inline operator/priceFSM.go]: 🔥🔥.

File: config-files/operator-1/raft.db
Comment (JdejesusIsaac) [reply config-files/operator-1/raft.db:1]: done.

**Review by Robert-H-Leonard — APPROVED** [review]

**Pull request merged**